alertmanager/config/config_test.go

```diff
@@ -8,17 +8,19 @@
 package config
 
 import (
+	"encoding/json"
+	"fmt"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
 var (
 	testConfig = Config{
-		Global:       nil,
-		Route:        &Route{
-			Receiver:       "base",
-			Routes:         []*Route{
+		Global: nil,
```

Comment:
Same

Reply:
This probably changed formatting because a lot of stuff was added to it, changing the spacing